app/assets/stylesheets/_mixins.scss

```diff
@@ -4,3 +4,12 @@
   letter-spacing: $letter-spacing-header;
   text-transform: uppercase;
 }
+
+@mixin link-target {
```

Comment:
When the span is `position: absolute` and a parent container is `position: <not static>` the span will start at the top left of that parent container. Add in the `top` and `left: 0` and the `width` and `height: 100%` and you have a span that is sitting exactly "over" the parent container. This expands the link to be the size of the container making it feel like you're clicking the entire container.

It needs a better name. `block-link` feels better.